pkg/database/vercode.go

```diff
@@ -75,36 +77,53 @@ func (v *VerificationCode) FormatSymptomDate() string {
 	return v.SymptomDate.Format("2006-01-02")
 }
 
+// IsCodeExpired checks to see if the actual code provides is the
+// short or long code and deteriminies if it is expired based on that.
+func (v *VerificationCode) IsCodeExpired(code string) bool {
+	now := time.Now()
+	if v.Code == code {
+		return !v.ExpiresAt.After(now)
+	}
+	return !v.LongExpiresAt.After(now)
```

Comment:
I understand what's going on here, but it's a little clever. Maybe:

```go
switch code {
  case v.Code:
    return !v.ExpiresAt.After(now)
  case v.LongCode:
    return !v.LongExpiresAt.After(now)
  default:
    return false
}
```

Reply:
either, way. changed it to switch, but the default should be true (expired)